commands/user.js

```diff
@@ -0,0 +1,132 @@
+/* jshint esversion: 11 */
+
+const {
+	SlashCommandBuilder,
+	ActionRowBuilder,
+	ButtonBuilder,
+	ButtonStyle
+} = require('discord.js');
+const config = require("../config/aod-discord-bot.config.json");
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('user')
+		.setDescription('Manage a user')
+		.addSubcommand(command => command.setName('kick').setDescription('Kicks a user from the server')
+			.addUserOption(option => option.setName('user').setDescription('User').setRequired(true))
+			.addStringOption(option => option.setName('reason').setDescription('Kick reason')))

```

Comment:
Probably worth making the reasons required for the slash commands

Reply:
Resolved in 7bbcdcc